ui/views_test.py

```diff
@@ -374,26 +374,6 @@ def test_program_thumbnail(self):
         resp = self.client.get('/')
         self.assertContains(resp, image.get_rendition('fill-690x530').url)
 
-    def test_courses_enrollment_text(self):
```

Comment:
I added a new test to verify this. Since we'll be switching away from rendering the course listing in HTML at all, I wrote a test that verifies that the courses are ordered correctly in the `SETTINGS` JS variable. Can you take a look, and tell me if it makes sense?